JLCoordinator/Sources/Presenter/PresenterImplementations/ModalNavigationPresenter.swift

```diff
@@ -69,6 +79,10 @@ public class ModalNavigationPresenter: ModalPresenting, NavigablePresenting {
 }
 
 extension ModalNavigationPresenter: AdaptivePresentationControllerDelegate {
+    func adaptivePresentationStyle(for controller: UIPresentationController) -> UIModalPresentationStyle {
+        return modalPresentationConfiguration.presentationStyle
```

Comment:
`Return` can be omitted

Reply:
I would like to keep the return statement to be more explicit.